electrum/gui/kivy/uix/dialogs/tx_dialog.py

```diff
@@ -250,6 +253,42 @@ def _do_rbf(
         self.update()
         self.do_sign()
 
+    def do_cpfp(self):
+        from .cpfp_dialog import CPFPDialog
+        parent_tx = self.tx
+        new_tx = self.wallet.cpfp(parent_tx, 0)
+        total_size = parent_tx.estimated_size() + new_tx.estimated_size()
+        parent_txid = parent_tx.txid()
+        assert parent_txid
+        parent_fee = self.wallet.get_tx_fee(parent_txid)
+        if parent_fee is None:
+            self.app.show_error(_("Can't CPFP: unknown fee for parent transaction."))
+            return
+        cb = partial(self._do_cpfp, parent_tx=parent_tx)
+        d = CPFPDialog(self.app, parent_fee, total_size, new_tx=new_tx, callback=cb)
+        d.open()
+
+    def _do_cpfp(
+            self,
+            fee,
+            max_fee,
+            is_final,
```

Comment:
Note that `is_final` is not used.
As said in another comment, let's just remove the checkbox from the UI and the related code.

```suggestion
```